packages/types/src/methods/send-transfer.ts

```diff
@@ -4,6 +4,7 @@ export interface SendTransferRequestParams {
   account?: number;
   address: string;
   amount: string;
+  network?: any;
```

Comment:
Should be this any? not `string`?

Reply:
Maybe something like `'testnet' | 'mainnet' | NetworkStruct` where

```ts
NetworkStruct {
  name: 'testnet' | 'mainnet',
  chainId: number
}
```

But could definitely start with a `'testnet' | 'mainnet'`